features/karaf/esh-core/src/main/feature/feature.xml

```diff
@@ -260,9 +259,19 @@
     <bundle>mvn:org.eclipse.smarthome.io/org.eclipse.smarthome.io.console.karaf/${project.version}</bundle>
   </feature>
 
-  <feature name="esh-io-rest-auth" version="${project.version}">
+  <feature name="esh-io-http-auth" version="${project.version}">
     <feature>esh-base</feature>
+    <bundle>mvn:org.eclipse.smarthome.io/org.eclipse.smarthome.io.http.auth/${project.version}</bundle>
+  </feature>
+
+  <feature name="esh-io-http-auth-basic" version="${project.version}">
+    <feature>esh-base</feature>
+    <feature>esh-io-http-auth</feature>
+    <bundle>mvn:org.eclipse.smarthome.io/org.eclipse.smarthome.io.http.auth.basic/${project.version}</bundle>
+  </feature>
```

Comment:
How about adding a
```
<config name="org.eclipse.smarthome.auth">
    enabled = true
</config>
```
here (in the light of https://github.com/eclipse/smarthome/pull/6351#issuecomment-429280159)?

Reply:
Done